### PR TITLE
add mariadb config to puppet

### PR DESCRIPTION
**Pull request by MacFan4000**

### CODE REVIEW

File: modules/mariadb/files/50-server.cnf [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+#slow_query_log_file    = /var/log/mysql/mariadb-slow.log
+#long_query_time        = 10
+#log_slow_verbosity     = query_plan,explain
+#log-queries-not-using-indexes
+#min_examined_row_limit = 1000
+
+# The following can be used as easy to replay backup logs or for replication.
+# note: if you are setting up a replication slave, see README.Debian about
+#       other settings you may need to change.
+#server-id              = 1
```
Comment (RhinosF1) [inline modules/mariadb/files/50-server.cnf:70]: Can we make this work so we can change it per server?

File: modules/mariadb/files/50-server.cnf [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+# Enable this if you want to have error logging into a separate file
+#log_error = /var/log/mysql/error.log
+# Enable the slow query log to see queries with especially long duration
+#slow_query_log_file    = /var/log/mysql/mariadb-slow.log
+#long_query_time        = 10
+#log_slow_verbosity     = query_plan,explain
+#log-queries-not-using-indexes
+#min_examined_row_limit = 1000
+
+# The following can be used as easy to replay backup logs or for replication.
```
Comment (RhinosF1) [inline modules/mariadb/files/50-server.cnf:67]: I think we can turn these on ready.

File: modules/mariadb/files/50-server.cnf [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+# note: if you are setting up a replication slave, see README.Debian about
+#       other settings you may need to change.
+#server-id              = 1
+#log_bin                = /var/log/mysql/mysql-bin.log
+expire_logs_days        = 10
```
Comment (RhinosF1) [inline modules/mariadb/files/50-server.cnf:72]: Probably only needs to be 2 at most

**Review by RhinosF1 — CHANGES_REQUESTED** [review]
Few comments left

File: modules/mariadb/files/50-server.cnf (hunk 1 above)
Comment (MacFan4000) [reply modules/mariadb/files/50-server.cnf:71]: I have no experience with replication, and don't know how best to handle this

**Pull request merged**